fhttp/http_test.go

```diff
@@ -80,6 +80,25 @@ func TestNewHTTPRequest(t *testing.T) {
 	}
 }
 
+func TestMultiInitAndEscape(t *testing.T) {
+	// one escaped already 2 not
+	o := NewHTTPOptions("http://localhost:8080/?delay=1s:10%,0.5s:15%25,0.25s:5%")
```

Comment:
suggestion:
expected := "http://localhost:8080/?delay=1s:10%,0.5s:15%25,0.25s:5%"
o := NewHTTPOptions(expected)

Reply:
expected is not same as input

Reply:
ok. it was hard to tell the difference.